Authenticator/Source/AppController.swift

```diff
@@ -43,8 +43,15 @@ class AppController {
     }()
 
     init() {
+        if Process.isDemo {
+            store = DemoTokenStore()
```

Comment:
I did something similar for use in the simulator since the keychain doesn't work. Might be worth enabling the mock token store when running in development mode on a simulator but without pre-configured tokens.